sklearn/model_selection/_validation.py

```diff
@@ -606,6 +614,9 @@ def cross_val_predict(estimator, X, y=None, groups=None, cv=None, n_jobs=1,
                       method='predict'):
     """Generate cross-validated estimates for each input data point
 
+    NB: It is not appropriate to pass these predictions into an evaluation
```

Comment:
I might prefer something other than ``NB`` (maybe ``Note that``?). I can't find similar word in our doc.

Reply:
If it is a note we should put it in a 

```
Notes
------
```

However, I think that this is more than a note. I would just remove `NB`.